src/azure-cli/azure/cli/command_modules/resource/tests/latest/test_resource.py

```diff
@@ -658,16 +658,17 @@ def applyPolicy(self):
         self.cmd('policy assignment delete -n {pan} -g {rg}')
         self.cmd('policy assignment list --disable-scope-strict-match', checks=self.check("length([?name=='{pan}'])", 0))
 
-    def applyPolicyAtScope(self, scope, policyId):
+    def applyPolicyAtScope(self, scope, policyId, enforcementMode='Default'):
```

Comment:
Might want to leave a test in place that doesn't provide enforcementMode them validates it correctly defaulted to Default.

Reply:
actually there are some tests that execute w/o enforcement mode that runs `applyPolicy` instead of `applicyPolicyAtScope`